build/Makefile

```diff
@@ -296,6 +296,8 @@ $(INTERMEDIATE_DIR)/$(OUTPUT_PACKAGE_PREFIX).tar : $(RUBY_DEST_DIR) $(IN_PLUGINS
 	cd $(DSC_DIR)/release; $(COPY) `ls omsconfig-*.ssl_100.*.deb | sort | tail -1` $(INTERMEDIATE_DIR)/100
 	cd $(INTERMEDIATE_DIR)/100; for f in omsconfig-*.{rpm,deb}; do SOURCE=$$f; DEST=`echo $$SOURCE | sed 's/.ssl_100././'` ; $(MV) $$SOURCE $$DEST; done
 
+	cd $(DSC_DIR)/release; $(COPY) `ls *.zip` $(TARGET_DIR)/
```

Comment:
Can you copy it into $(TARGET_DIR)/$(ARCH)/? It will help us to find the packages and merge them in build runner.